package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "resin-typescript-skeleton",
+    "version": "0.0.1",
+    "description": "Skeleton template for a Resin TypeScript project",
+    "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+    "types": "build/index.d.ts",
```

Comment:
Sorry, I don't have any hard answers here. I've always seen declaration files used for `types`, but if the non-declaration file approach works the same too, but with fewer files and less config, that sounds great to me.

Personally, I'd probably go for @CameronDiver's approach for now. If we do find there's a good reason why this won't quite work then it isn't very hard to change later, and otherwise I do agree it's cleaner.